src/net.h

```diff
@@ -294,6 +295,8 @@ class CConnman
     void SetBestHeight(int height);
     int GetBestHeight() const;
 
+    /** Get a unique deterministic randomizer. */
+    CSipHasher GetDeterministicRandomizer(uint64_t id);
```

Comment:
Well it's only used for randomization that is specific to networking. The chainstate cache for example also has a randomized, and that one is independent.

I expect longer term that there will be something like network event/message handlers, and the ProcessMessage code in main now will move to such a handler that registers itself with a connman. The randomizer code could then go into the specific network handler instance, instead of with connman itself.